client/src/app/modules/events/event-edit/event-edit.component.html

```diff
@@ -1,53 +1,160 @@
+          <div class="col-md-6">
+            <div class="form-group">
+              <label for="dateEnd" class="form-control-label" [translate]="'DATE_END'"></label>&nbsp;
+              <span class="error-message" 
+                    *ngIf="eventForm.controls.dateEnd.invalid" 
+                    [translate]="'ERROR_REQUIRED_FIELD'"></span>
+              <input type="date" 
+                    class="form-control" 
+                    [ngClass]="{'error' : eventForm.controls.dateEnd.invalid}"
+                    id="dateEnd" 
+                    formControlName="dateEnd">
+            </div>
           </div>
         </div>
-        <div class="row form-group">
-          <label for="location" class="form-control-label" [translate]="'LOCATION'"></label>
-          <input type="text" class="form-control" id="location" placeholder="Ševčenkova 34" [(ngModel)]="event.location" name="location">
+        <div class="row">
+          <div class="col">
+            <div class="form-group">
+              <label for="location" class="form-control-label" [translate]="'LOCATION'"></label>&nbsp;
+              <span class="error-message" 
+                    *ngIf="eventForm.controls.location.invalid" 
+                    [translate]="'ERROR_REQUIRED_FIELD'"></span>
+              <input type="text" 
+                    class="form-control" 
+                    [ngClass]="{'error' : eventForm.controls.location.invalid}"
+                    id="location" 
+                    placeholder="Ševčenkova 34" 
+                    formControlName="location">
+            </div>
+          </div>
         </div>
-        <div class="row form-group">
-          <label for="formDescInput" [translate]="'DESCRIPTION'"></label>
-          <textarea type="text" class="form-control" id="formDescInput" rows="6" placeholder="{{ 'INSERT_EVENT_DESCRIPTION' | translate }}"
-            name="description" [(ngModel)]="event.description"></textarea>
+        <div class="row">
+          <div class="col">
+            <div class="form-group">
+              <label for="description" class="form-control-label" [translate]="'DESCRIPTION'"></label>
+              <textarea type="text" class="form-control" id="description" rows="6" placeholder="{{ 'INSERT_EVENT_DESCRIPTION' | translate }}" formControlName="description"></textarea>
+            </div>
+          </div>
         </div>
-      </div>
-    </div>
-    <div class="col-md-4 pr-0 pl-0" id="atendeeList">
-      <div class="container">
-        <div class="row justify-content-between mb-2 align-items-end">
-          <div class="col-md-4 pl-0">
-            <span class="form-control-label" [translate]="'ATTENDEES'"></span>
+        <div class="row">
+          <div class="col">
+            <div class="form-group">
+              <label for="participantsLimit" class="form-control-label" [translate]="'PARTICIPANTS_LIMIT'"></label>
+              <input type="number" 
```

Comment:
It will allow negative numbers as well... by using `min="0"` less than 0 won't be allowed to pick

Reply:
good point, I'll add validator for this